test/tokenomics/rootGaugeLocal.ts

```diff
@@ -112,9 +118,6 @@ describe("Root Gauge (Local)", () => {
       )
 
       // If rate is not initialized, initialize it
-      if ((await minter.rate()).eq(MAX_UINT256)) {
-        await minter.update_mining_parameters()
-      }
```

Comment:
Removed because already updated in `veSDL` fixture?

Reply:
Yep